apps/client/src/features/operator/operator-event/OperatorEvent.tsx

```diff
@@ -89,12 +89,19 @@ function OperatorEvent(props: OperatorEventProps) {
       </span>
 
       <div className={style.fields}>
-        {subscribed && (
-          <>
-            <span className={style.field}>{subscribedAlias}</span>
-            <span className={style.value}>{subscribed}</span>
-          </>
-        )}
+        {subscribed &&
+          subscribed
+            .filter((field) => field.value)
+            .map((field, index) => (
+              <div key={index}>
```

Comment:
tip: it is good to avoid using indexes as keys since they dont accurately distinguish the dataset (example: if the dataset reorders, a different piece of data can be at this dataset)

```suggestion
            .map((field) => (
              <div key={field.id}>
```